test/player-test.js

```diff
@@ -2,32 +2,36 @@ import chai from 'chai';
 const expect = chai.expect;
 
 const Player = require('../src/Player')
-
+const Game = require('../src/Game')
 
 describe('Player', () => {
-  let player;
-  
+  let player1, player2, player3;
+  let game
+
   beforeEach(() => {
-    player = new Player('Billy')
+    player1 = new Player('Carla')
+    player2 = new Player('Vee')
+    player3 = new Player('Novak')
+    game = new Game([player1, player2, player3])
   });
 
   it('should be an instance of a player', () => {
-    expect(player).to.be.an.instanceOf(Player)
+    expect(player1).to.be.an.instanceOf(Player)
   });
 
   it('should have a name', () => {
-    expect(player.name).to.equal('Billy')
+    expect(player1.name).to.equal('Carla')
   });
 
   it('should know whether it\'s their turn or not', () => {
-    expect(player.turn).to.equal(false)
+    expect(player1.turn).to.equal(false)
```

Comment:
I know this branch was closed in favor of a different one, but make sure you have a test for the happy path of this property (player knowing that it is their turn)